### PR TITLE
Update changelogs and deps

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,15 +4,98 @@
...
+* Support quiesce/dequiesce of routers for auto-scaling and/or round-robin upgrades
+
+## Quiesce/Dequiesce Routers
+
+The ziti CLI supports two new operations:
+
+* `ziti agent router quiesce`
+* `ziti agent router dequiesce`
+
+When told to quiesce, the targeted router will reach out the controller and request that its terminators be marked as
```
Comment (scareything) [inline CHANGELOG.md:16]: reach out _to_ the controller?

Comment (plorenz) [reply CHANGELOG.md]: fixed, thank you

**Review by scareything — APPROVED** [review]
lgtm. I'm sure you'll sync with release-next before merging.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,15 +4,98 @@
...
+* github.com/openziti/metrics: [v1.2.25 -> v1.2.26](https://github.com/openziti/metrics/compare/v1.2.25...v1.2.26)
 * github.com/openziti/ziti: [v0.28.0 -> v0.28.1](https://github.com/openziti/ziti/compare/v0.28.0...v0.28.1)
   * [Issue #1132](https://github.com/openziti/ziti/issues/1132) - Updated `ws` protocol to `wss` as `ws` is no longer supported.
-* github.com/openziti/fabric: [v0.23.29 -> v0.23.35](https://github.com/openziti/fabric/compare/v0.23.29...v0.23.35)  
-  * [Issue #732](https://github.com/openziti/fabric/issues/732) - 
-  Added new `bufferSize` config option to amqp handler. Connection handling now happens in the background with exponential retries.
-* github.com/openziti/edge: [v0.24.309 -> v0.24.326](https://github.com/openziti/edge/compare/v0.24.309...v0.24.326)
-  * [Issue #1517](https://github.com/openziti/edge/issues/1517) - allow wildcard domains in intercept.v1 addresses
 
 ### AMPQ Event Writer Changes
```
Comment (scareything) [inline CHANGELOG.md:100]: Also I see that I missed an "AMQP" typo here. Could you get it?

Comment (plorenz) [reply CHANGELOG.md]: fixed

**Comment by plorenz** [comment]
> lgtm. I'm sure you'll sync with release-next before merging.

rebased :+1:

**Pull request merged**